modules/installation-configuration-parameters.adoc

```diff
@@ -929,7 +929,11 @@ Additional Azure configuration parameters are described in the following table:
 
 |`controlPlane.platform.azure.osDisk.diskSizeGB`
 |The Azure disk size for the VM.
-|Integer that represents the size of the disk in GB. The minimum supported disk size is `1024`.
+|Integer that represents the size of the disk in GB. The minimum supported disk size is `1024
+
+|`controlPlane.platform.azure.osDisk`
+|`diskType` defines the type of disk
+|`standard_LRS`, `premium_LRS`, `standardSSD_LRS`. For control panel nodes: `premium_LRS`. `standardSSD_LRS`. Default is `premium_LRS`.
```

Comment:
```suggestion
|`standard_LRS`, `premium_LRS`, `standardSSD_LRS`. For control panel nodes: `premium_LRS`. `standardSSD_LRS`. The default is `premium_LRS`.
```

I am confused about the list of values. The name of the parameter makes me think that this is just for control plane nodes, but the description limits the values available to the control plane. When do you use the other ones?

Reply:
one is for controPanel, the other is for compute, 
controlPlane.platform.azure.osDisk.**diskSizeGB** has the same issue, 
actually if not provide the structure as https://github.com/openshift/installer/blob/master/docs/user/azure/customization.md#custom-machine-pools, hard to know how to specify them 
how about use same row for controPanel.platform.azure.osDisk  and  compute.platform.azure.osDisk

Reply:
I just made it so the column shows both, do you think that helps with the confusion?